apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/success/page.tsx

```diff
@@ -149,11 +160,8 @@ export default async function CheckoutSuccess({
           )}
         />
 
-        <a
-          className="page-button"
-          href={cms.commonContent.successActionButtonUrl}
-        >
-          {cms.commonContent.successActionButtonLabel ||
+        <a className="page-button" href={successActionButtonUrl}>
+          {successActionButtonUrl ||
```

Comment:
L163 looks good, but L164 should reflect the label/text on the button (`successActionButtonLabel`) rather than the url (`successActionButtonUrl`).